CONTRIBUTING.md

```diff
@@ -1,71 +1,112 @@
-   [dwave-examples/template-dash](https://github.com/dwave-examples/template-dash) repository with a link to your example.
-
-4. To add your example, D-Wave will fork your repository. We may make a pull
-   request if we would like your approval for certain code changes.
-
-## Guidelines
-
-### Files
-
-Please make sure that your example includes the following:
-
-* `README.md`:
-    * Should be written in Markdown (and not reStructuredText).
-    * See this repository's [README](README.md).
-
-* `requirements.txt`:
-    * `dwave-ocean-sdk` should be unpinned and lower bounded.
-    * Add requirements for running your example and make sure they are either pinned to a version supported by the latest Ocean SDK release or lower bounded.
-
-* `LICENSE`:
-    * Examples should comply with the Apache 2.0 License. Please make sure that:
-        * All source and test files include a license header.
-        * The [`LICENSE`](LICENSE) file is included in your root directory.
-
-* Tests that are located in `/tests/` and discoverable through `python -m unittest discover`.
-
-Our examples are tested using CircleCI. For a list of operating systems and
+1.  Follow GitHub's instructions for
+    [creating a repository from a template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template).
+    Alternatively, clone this repository as a starting point from which to implement your example.
+
+2.  Implement your example by following these instructions:
+
+    1.  It is preferable that all Python and CSS code occur either in new files or in files that
+        contain `demo` in the name. Below is a list of `demo` files and their functionality:
+
+        *   [demo_callbacks.py](demo_callbacks.py) contains all the
+            [Dash callback functions](https://dash.plotly.com/basic-callbacks) required to run the
+            Dash app. Updates to existing callbacks and new callbacks should be made in this file.
+        *   [demo_interface.py](demo_interface.py) contains all the Dash HTML components that
+            generate the user interface (UI) for the app, including settings like sliders,
+            checkboxes, and text inputs, as well as buttons and tables. If a Dash HTML component
+            needs to be dynamically added to the UI by a Dash callback in `demo_callbacks.py`, it
+            is encouraged to add a function generating this component to `demo_interface.py` and
+            call this function from `demo_callbacks.py`.
+        *   [demo_configs.py](demo_configs.py) contains all configurations and settings for the
+            demo and is intended as a quick way to customize the demo for a specific audience or
+            use case. It is encouraged to add more customization settings to this file; however,
+            this file should not be cluttered with settings that are not helpful to the end user
+            (eg titles and other settings that are unlikely to ever be changed).
```

Comment:
```suggestion
        *   [demo_configs.py](demo_configs.py) contains all configurations and settings for the
            demo and is intended as a quick way to customize the demo for a specific audience or
            use case. Add settings to this file that users might want to change; for example, ranges
            of inputs and number of selections but not component names and internal variables.
```

Just some alternate phrasing you can look at and maybe use

Reply:
changed this slightly but it's the same idea https://github.com/dwave-examples/template-dash/pull/3/commits/bb2bff29e7c4d282ef3e67a8f4582e0bc7ae9d75